tests/ARCValidationPackages.Tests/ReferenceObjects.fs

```diff
@@ -52,51 +55,73 @@ validationCases
 )"""                        .ReplaceLineEndings()
 
 let testValidationPackage1 =
-    ARCValidationPackage.create(
+    CachedValidationPackage.create(
         "test@1.0.0.fsx",
         testDate1,
-        (Path.Combine(expected_package_cache_folder_path, "test@1.0.0.fsx").Replace("\\","/")),
-        ValidationPackageMetadata.create("test", "this package is here for testing purposes only.", 1, 0, 0)
+        (Path.Combine(expected_package_cache_folder_path_preview, "test@1.0.0.fsx").Replace("\\","/")),
+        ValidationPackageMetadata.create("test", "this package is here for testing purposes only.", "this package is here for testing purposes only.", 1, 0, 0)
     )
 
 let testValidationPackage2 =
-    ARCValidationPackage.create(
+    CachedValidationPackage.create(
         "test@1.0.0.fsx",
         testDate2,
-        (Path.Combine(expected_package_cache_folder_path, "test@1.0.0.fsx").Replace("\\","/")),
-        ValidationPackageMetadata.create("test", "this package is here for testing purposes only.", 1, 0, 0)
+        (Path.Combine(expected_package_cache_folder_path_preview, "test@1.0.0.fsx").Replace("\\","/")),
+        ValidationPackageMetadata.create("test", "this package is here for testing purposes only.", "this package is here for testing purposes only.", 1, 0, 0)
     )
 
 let testValidationPackage3FullMetadata =
-    ARCValidationPackage.create(
+    CachedValidationPackage.create(
         "test@3.0.0.fsx",
         testDate3,
-        (Path.Combine(expected_package_cache_folder_path, "test@3.0.0.fsx").Replace("\\","/")),
+        (Path.Combine(expected_package_cache_folder_path_preview, "test@3.0.0.fsx").Replace("\\","/")),
         ValidationPackageMetadata.create(
-            "test", 
+            "test",
+            "this package is here for testing purposes only.",
             "this package is here for testing purposes only.", 
             1, 
             0, 
             0,
             Publish = true,
             Authors = [|
-                Author.create(
-                    fullName = "John Doe",
-                    email = "j@d.com",
-                    Affiliation = "University of Nowhere",
-                    AffiliationLink = "https://nowhere.edu"
+                let author1 = new Author()
+                let author2 = new Author()
```

Comment:
You'll have `Author.create` with Index package

Reply:
yes i know, this was from before those functions were included